docs/examples/can_lin_diff.rst

```diff
@@ -0,0 +1,20 @@
+Adapting CAN examples to LIN
+============================
+
+This displays the diff of ``can_frame_stream_io.py`` and 
+``lin_frame_stream_io.py`` to demonstrate the changes required to
+update CAN example code for LIN.
+
+To adapt this example to LIN frames, reference signals in a database that use LIN:
```

Comment:
Yep, I was not understanding. Thanks for clearing that up @epage. Added these in my latest commit. Sorry for the oversights!